index.html

```diff
@@ -8,16 +8,16 @@
   <link rel="stylesheet" href="style.css">
   <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
 </head>
-<body>
   <header>
     <a href="#"><h2>John Doe</h2></a>
     <nav>
-      <img src="menu.png" class="menu" alt="">
-      <ul class="menu_items">
-        <li>Portfolio</li>
-        <li>About</li>
-        <li>Contact</li>
-        <li><img src="mail.png" class="mail" alt="image"></li>
+      <button type="button" id="toggleBtn"><img src="/images/menu.png" class="menu" alt=""></button>
+      <ul class="menu_items" id="menuList">
+        <li><button type="button" id="btn_close">X</button></li>
+        <li><a href="#project-s">Portfolio</a><span>></span></li>
+        <li><a href="#headline">About</a><span>></span></li>
+        <li><a href="#contact-me">Contact</a><span>></span></li>
+        <li><img src="/images/mail.png" class="mail" alt="image"><span>></span></li>
       </ul>
```

Comment:
- [ ] OPTIONAL
Your hamburger menu and the close button are not in the same position when switched. Your close button appears further left than the hamburger menu. Following the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10), let's ensure that the hamburger and the close button are on the same position, and doing so will enhance great user experience.